mmv1/loader/loader.go

```diff
@@ -293,17 +285,17 @@ func (l *Loader) loadResource(product *api.Product, baseResourcePath string, ove
 	if overrideResourcePath != "" {
 		if baseResourceExists {
 			// Merge base and override
-			api.Compile(baseResourcePath, resource, l.OverrideDirectory)
+			api.Compile(baseResourcePath, resource)
 			overrideResource := &api.Resource{}
-			api.Compile(overrideResourcePath, overrideResource, l.OverrideDirectory)
+			api.Compile(overrideResourcePath, overrideResource)
 			api.Merge(reflect.ValueOf(resource).Elem(), reflect.ValueOf(*overrideResource), l.Version)
 		} else {
 			// Override only
-			api.Compile(overrideResourcePath, resource, l.OverrideDirectory)
+			api.Compile(overrideResourcePath, resource)
 		}
 	} else {
 		// Base only
-		api.Compile(baseResourcePath, resource, l.OverrideDirectory)
+		api.Compile(baseResourcePath, resource)
 		resource.SourceYamlFile = baseResourcePath
```

Comment:
```suggestion
		resource.SourceYamlFile = filepath.Rel(baseResourcePath)
```

you might need to do this so that the file reference headers stays the same. Note that I don't know if code above is correct, only guessing at the interface.

Reply:
yeah headers need some work. i am thinking if it is not a relative path and an absolute path that the header lists a fs location instead of github location. Alternatively a -production- flag which helps. regardless I think its fine as is as long as we don't have a diff.